CHANGELOG

```diff
@@ -100,6 +100,7 @@
 
 == Version 1.114.0
 * BlueSnap: Add address1,address2,phone,shipping_* support #3749
+* Netbanx: Add-customer-information(name,email,IP)-to-a-transaction [rockyhakjoong] #3754
```

Comment:
Can you add that to the `=== HEAD` section

Reply:
I updated it. thank you.